cogs/rockpaperscissors.py

```diff
@@ -0,0 +1,30 @@
+import nextcord
+from nextcord.ext import commands
+import random
+
+class rps(commands.Cog):
```

Comment:
Class names should be capitalized
```suggestion
class RPS(commands.Cog):
```